src/promptflow-devkit/promptflow/_sdk/entities/_run.py

```diff
@@ -196,6 +196,9 @@ def __init__(
         self._identity = kwargs.get("identity", {})
         self._outputs = kwargs.get("outputs", None)
         self._command = kwargs.get("command", None)
+        # To support `pf.run` for dynamic callable, we need to create a run whose target function is a dynamic callable.
+        # TODO: such run is not resumable, not sure if we need specific error message for this case.
```

Comment:
I think we can add a properties for this actually

Reply:
You mean make it public? or add it to `__init__`? Here my concern, this property will be lost when we do `pf.runs.get`, so it will be strange if we make it a public attribute - it's hard for customer to figure out when this property shouldn't be `None`.

Reply:
And I don't think it's a good idea to add this argument to `__init__` if we don't give it a public `attribute`. Or you mean we can make it a private attribute first?